templates/default.html

```diff
@@ -2,10 +2,16 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Resin.io Documentation</title>
+    <title>{{ title }} - Resin.io Documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" type="image/x-icon" href="/favicon.png" />
 
+    <meta property="fb:app_id" content="221218511385682" />
```

Comment:
I'd better move this to config/index.coffee (define a constant + export it in`layoutLocals`)